### ISSUE TITLE
iptorrents broken 

### ISSUE THREAD

**Issue by icepicknz** [SickChill/SickChill#6372]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: Latest, just did latest commit right now and same issue
OS: Centos
Browser: Safari or Firefox
What you did: Trying to download using iptorrents and forcing backlog
What happened: IPTorrents won't work because of cloudflare. Been broken 7 days or so, reported to the GitHub as suggested but this looks like it may be another issue, problem looks to have started after I did a major commit update
What you expected: iptorrents to work
Logs:
```
2020-05-04 21:35:10 INFO SEARCHQUEUE-BACKLOG- :: No needed episodes found during backlog search for: [The Amazing Race Australia] 
2020-05-04 21:35:10 INFO SEARCHQUEUE-BACKLOG-248618 :: [IPTorrents] :: No NZB/Torrent providers found or enabled in the sickchill config for backlog searches. Please check your settings. 
AA AttributeError: 'exceptions.ValueError' object has no attribute 'request' 
AA if requests_exception.request: 
AA File "/opt/sickrage/sickbeard/helpers.py", line 1467, in handle_requests_exception 
AA handle_requests_exception(error) 
AA File "/opt/sickrage/sickbeard/helpers.py", line 1362, in getURL 
AA return getURL(url, post_data, params, self.headers, timeout, self.session, **kwargs) 
AA File "/opt/sickrage/sickchill/providers/GenericProvider.py", line 381, in get_url 
AA File "/opt/sickrage/sickbeard/providers/iptorrents.py", line 86, in login 
AA results = [] 
AA File "/opt/sickrage/sickbeard/providers/iptorrents.py", line 111, in search 
AA items_list += self.search(search_string, ep_obj=episode) 
AA File "/opt/sickrage/sickchill/providers/GenericProvider.py", line 177, in find_search_results 
AA searchResults = curProvider.find_search_results(show, episodes, search_mode, manualSearch, downCurQuality) 
AA File "/opt/sickrage/sickbeard/search.py", line 487, in searchProviders 
AA Traceback (most recent call last):
2020-05-04 21:35:10 ERROR SEARCHQUEUE-BACKLOG-248618 :: [IPTorrents] :: [c5ed827] Exception while searching IPTorrents. Error: AttributeError("'exceptions.ValueError' object has no attribute 'request'",) 
AA Please read https://github.com/Anorov/cloudflare-scrape#updates, then file a bug report at https://github.com/Anorov/cloudflare-scrape/issues." 
AA ValueError: Unable to identify Cloudflare IUAM Javascript on website. Cloudflare may have changed their technique, or there may be a bug in the script. 
AA raise requests_exception 
AA File "/opt/sickrage/sickbeard/helpers.py", line 1423, in handle_requests_exception 
AA Traceback (most recent call last): AA Please read https://github.com/Anorov/cloudflare-scrape#updates, then file a bug report at https://github.com/Anorov/cloudflare-scrape/issues." () 
2020-05-04 21:35:10 ERROR SEARCHQUEUE-BACKLOG-248618 :: [IPTorrents] :: [c5ed827] Request failed: Unable to identify Cloudflare IUAM Javascript on website. Cloudflare may have changed their technique, or there may be a bug in the script.
 ```

**Comment by VeNoMouS** [comment]
@miigotu just a heads up... if i actually publicly publish the fix for the latest Cloudflare you wont be able to do it on py2 due to the cpython code bindings... just saying 

I did publish a fix yesterday but cloudflare patched within 30 minutes and their CEO tweeted my github issue therad.

**Comment by miigotu** [comment]
@VeNoMouS is there any way to back port it? I could try and find a way to include the code privately maybe?

**Comment by VeNoMouS** [comment]
Due to the limitations of python, i haven't worked out a way yet without modifying python source code

**Comment by VeNoMouS** [comment]
Actually, let me check something

**Comment by VeNoMouS** [comment]
Ok i just found another way to bypass

**Comment by miigotu** [comment]
Seems as though they are monitoring your repo though...

**Comment by VeNoMouS** [comment]
Don't get me wrong, for the most part, 1.2.36 will work fine, unless the site has higher security turned on, ie www.sneakersnstuff.com the only way to get around that is run my private version of the bypass

**Comment by miigotu** [comment]
@VeNoMouS a lot of them are turning it on, torrentz2,eu, iptorrents, yggtorrent for example. We are down quite a few trackers right now bc of cloudflare

**Comment by hooligeek** [comment]
been following this thread because I also used IPT.  I have a work around in place that may offer some insight into a solution.  I have Jackett configured as a proxy.  Currently SC is still able to reach IPT through the Jackett proxy.

`2020-05-08 10:10:46 INFO SEARCHQUEUE-DAILY-SEARCH :: Saved result to T:\Torrents\Harley.Quinn.S02E06.All.the.Best.Inmates.Have.Daddy.Issues.720p.DCU.WEB-DL.DDP5.1.H264-NTb.torrent
2020-05-08 10:10:46 INFO SEARCHQUEUE-DAILY-SEARCH :: Downloading a result from Jackett at http://127.0.0.1:9117/dl/iptorrents/?jackett_apikey=**********&path=Q2ZESjhBYlR4TGQzNkFKRm1DZWNVdVM5NlM5YTM3VlQ4VUVfYzBLMVA1ZzNfSkRhMGs2Q3RQejNaYjBoVUJLSXFoWGcxOWFPVUZzUDQ2YnI4S3MwVUFnZlo5TENYTjVHYU42ZlJFMDc2X2t5TGtEekFRUGM3RlN6UTBYU1kwRXRHTktDZzRiaDVFcDRESHdEeUplZlJ3Z0h5ajFKQTA2Q1U1Vmh0U01aSWpoYmtPT2NpNkJ4WlROMWtIQkZLRl8wTU9xZVF4ci1BQm42UVNEams2QlJlNXZGdTViR29RS2daNVFHSzB1YkZBRU9NdGdjeWtudkQ0YUZLSEltQldkNVJhd0tIbjJZX1ZiMWVFcUJyNkZtRFlUdHdLNUZKWkJQNlU3RzVxUEhwR0hud2Job1RCdGlFWUdOakxTLTNJaHlxTmNjZWc&file=Harley+Quinn+S02E06+All+the+Best+Inmates+Have+Daddy+Issues+720p+DCU+WEB-DL+DDP5+1+H264-NTb
2020-05-08 10:10:45 INFO SEARCHQUEUE-DAILY-SEARCH :: Downloading Harley.Quinn.S02E06.All.the.Best.Inmates.Have.Daddy.Issues.720p.DCU.WEB-DL.DDP5.1.H264-NTb from Jackett
2020-05-08 10:10:45 INFO SEARCHQUEUE-DAILY-SEARCH :: [Jackett] :: Quality of Harley.Quinn.S02E06.All.the.Best.Inmates.Have.Daddy.Issues.720p.DCU.WEB-DL.DDP5.1.H264-NTb is 720p WEB-DL
2020-05-08 10:10:45 INFO SEARCHQUEUE-DAILY-SEARCH :: [Jackett] :: Found result Harley.Quinn.S02E06.All.the.Best.Inmates.Have.Daddy.Issues.720p.DCU.WEB-DL.DDP5.1.H264-NTb at http://127.0.0.1:9117/dl/iptorrents/?jackett_apikey=**********&path=Q2ZESjhBYlR4TGQzNkFKRm1DZWNVdVM5NlM5YTM3VlQ4VUVfYzBLMVA1ZzNfSkRhMGs2Q3RQejNaYjBoVUJLSXFoWGcxOWFPVUZzUDQ2YnI4S3MwVUFnZlo5TENYTjVHYU42ZlJFMDc2X2t5TGtEekFRUGM3RlN6UTBYU1kwRXRHTktDZzRiaDVFcDRESHdEeUplZlJ3Z0h5ajFKQTA2Q1U1Vmh0U01aSWpoYmtPT2NpNkJ4WlROMWtIQkZLRl8wTU9xZVF4ci1BQm42UVNEams2QlJlNXZGdTViR29RS2daNVFHSzB1YkZBRU9NdGdjeWtudkQ0YUZLSEltQldkNVJhd0tIbjJZX1ZiMWVFcUJyNkZtRFlUdHdLNUZKWkJQNlU3RzVxUEhwR0hud2Job1RCdGlFWUdOakxTLTNJaHlxTmNjZWc&file=Harley+Quinn+S02E06+All+the+Best+Inmates+Have+Daddy+Issues+720p+DCU+WEB-DL+DDP5+1+H264-NTb
2020-05-08 10:10:24 INFO SEARCHQUEUE-DAILY-SEARCH :: Beginning daily search for new episodes`

Maybe helpful, maybe not but figured I'd share.

-HG
